lib/modules/manager/nuget/update.ts

```diff
@@ -30,7 +31,19 @@ export function bumpPackageVersion(
 
   try {
     const project = new XmlDocument(content);
-    const versionNode = project.descendantWithPath('PropertyGroup.Version')!;
+    const versionNode = findVersion(project);
+    if (!versionNode) {
+      logger.warn(
+        "Couldn't find Version or VersionPrefix in any PropertyGroup"
+      );
+      return { bumpedContent };
+    }
+    if (versionNode.val !== currentValue) {
```

Comment:
it seems you had a wrong merge from main

Reply:
If you think there's something wrong here, I'll fix it, but I don't see a mis-merge.

Reply:
your PR is conflicted, needs deconflicting